src/nrnpython/CMakeLists.txt

```diff
@@ -225,6 +225,9 @@ if(NRN_ENABLE_MODULE_INSTALL)
             ${PROJECT_BINARY_DIR}/share/nrn/lib
     COMMAND ${CMAKE_COMMAND} -E copy_directory ${PROJECT_SOURCE_DIR}/share/demo
             ${PROJECT_BINARY_DIR}/share/nrn/demo)
+    
+  file(COPY ${PROJECT_SOURCE_DIR}/share/lib/python/neuron/help_data.dat
+        DESTINATION ${PROJECT_BINARY_DIR}/lib/python/neuron/)
```

Comment:
Just curious - what is failing? I wonder how this was not detected earlier?

Reply:
I suspect I am installing in a different path. But that file has to be there for testing purposes

Reply:
Interesting. @neuroman314 complained about  a missing help_data.dat file a few days ago. I wonder if  this will fix that issue for his situation.

Reply:
was he using miniconda?

Reply:
Maybe a different issue altogether. He experienced
```
>>> help(h.IClamp)
...
FileNotFoundError: [Errno 2] No such file or directory:
'/usr/local/lib64/python3.6/site-packages/neuron/help_data.dat'
```
but he did have that file in
```
/usr/local/lib/python3.6/site-packages/neuron/help_data.dat
```
and he mentions another (now that I think about it, somewhat contradictory) clue
> On my machine there is no /usr/local/lib/python3.6/site-packages
but there is a /usr/local/lib/python3.6/dist-packages containing installed modules such as matplotlib, pandas, etc.

maybe not worth pursuing as we finished off with
> python3.6 -m pip  install --user neuron-nightly
...
works

Reply:
Needs to be looked at `/usr/local/lib64` vs `/usr/local/lib` aspects separately. I wonder if @ferdonline has any memories about how packages decide to install in one or another.